src/oas3/guards.ts

```diff
@@ -50,3 +51,10 @@ export const isTagObject = (maybeTagObject: unknown): maybeTagObject is TagObjec
 
   return false;
 };
+
+export const isMediaObject = (maybeMediaObject: unknown): maybeMediaObject is MediaTypeObject =>
+  isObject(maybeMediaObject) &&
+  ('schema' in maybeMediaObject ||
+    'examples' in maybeMediaObject ||
+    'example' in maybeMediaObject ||
+    'encoding' in maybeMediaObject);
```

Comment:
According to @stoplight/types "schema", "examples", and "encodings" are all optional on the media type content interface. So we probably shouldn't require them here, right?

https://github.com/stoplightio/types/blob/master/src/http-spec.ts#L128-L136

Reply:
@lottamus is right, according to the spec the Media Object has no required field — making fundamentally *anything* a valid `MediaObject`

Reply:
Yeah, that's right, all properties are optional.
That being said, if none of them is present, we won't produce anything meaningful anyway.